hyperspy/tests/mva/test_bss.py

```diff
@@ -120,10 +120,14 @@ def test_diff_axes_string_with_mask(self):
         mask.fold()
         self.s.learning_results.factors[5, :] = np.nan
         factors = self.s.get_decomposition_factors().inav[:3]
+        mask.change_dtype("float")
+        mask_diff = mask.diff(axis="x", order=1)
+        mask.change_dtype("bool")
+        mask_diff.change_dtype("bool")
         self.s.blind_source_separation(
             3, diff_order=0, fun="exp", on_loadings=False,
             factors=factors.diff(axis="x", order=1),
-            mask=mask.diff(axis="x", order=1))
```

Comment:
Yes, it works with 1.13.1, but not 1.13.0... Considering numpy can be annoying to install from source, should we leave as it is or exclude 1.13.0 but keep the code more simple?